tests/integration/targets/ndo_l3_domain/tasks/main.yml

```diff
@@ -85,20 +83,20 @@
           - cm_create_new_l3_domain is changed
           - cm_create_new_l3_domain.previous == {}
           - cm_create_new_l3_domain.current.name == "ansible_test_l3_domain"
-          - cm_create_new_l3_domain.current.pool == "ansible_test_vlan_pool_1"
+          - cm_create_new_l3_domain.current.pool != ""
```

Comment:
so we only return the pool uuid?

Reply:
Yes, let me insert the name attribute into the output.